README.md

```diff
@@ -282,6 +284,8 @@ Define your server connections in a JSON file:
 }
 ```
 
+Note: For stdio connections, the `transport` field is optional. If not specified, it defaults to 'stdio'.
```

Comment:
consider changing to this style to make the callout more prominent https://github.com/orgs/community/discussions/16925